Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/OverworldUI.cs

```diff
@@ -180,6 +185,16 @@ public void Disable()
 		SwitchFocus (_CommanderUI);
 	}
 
+	public void _ArmouryUI_OnCurrencyChanged(int val, Player player)
+	{
+		_ArmouryUI.CurrencyChangedUpdate(val, player);
+	}
+
+	public void _ArmouryUI_OnShowToggled(bool toggledOn, Player player)
```

Comment:
Yes, it's because I had it as listener before and forgot to change. Removed it now. I disagree about separate two methods instead of one with a bool :P